Tools/jmavsim_run.sh

```diff
@@ -12,6 +12,7 @@ device=
 ip="127.0.0.1"
 protocol="tcp"
 while getopts ":b:d:u:p:qsr:f:i:loat" opt; do
+#while getopts "b:d:up:qsr:i:loat" opt; do
```

Comment:
I think the changes you have proposed are commented out :smile:

Reply:
Sorry, 
I found my mistake and I want to fix it ASAP before you check it.  :-(

Reply:
I fixed my mistake :-) 
Please check it again.  (Yes, I want to update it because of "-f" , "-p" options